helfi_platform_config.module

```diff
@@ -39,12 +40,36 @@ function helfi_platform_config_modules_installed(array $modules, bool $is_syncin
   helfi_platform_config_update_paragraph_target_types();
 }
 
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function helfi_platform_config_base_field_override_presave(
+ BaseFieldOverride $field
+) : void {
+  if ($field->get('entity_type') !== 'paragraphs_library_item') {
+    return;
+  }
+  helfi_platform_config_rebuild_handler_settings($field);
+}
+
 /**
  * Implements hook_ENTITY_TYPE_presave().
  */
 function helfi_platform_config_field_config_presave(
   FieldConfigInterface $field
 ) : void {
+  helfi_platform_config_rebuild_handler_settings($field);
+}
+
+/**
+ * Rebuild field handler settings target bundles after saving fields.
+ *
+ * @param BaseFieldOverride|FieldConfigInterface $field
+ *   Base field override or field config.
+ */
+function helfi_platform_config_rebuild_handler_settings(
+  BaseFieldOverride|FieldConfigInterface $field
```

Comment:
BaseFieldOverride implements FieldConfigInterface, so I assume BaseFieldOverride type hint is unnecessary